easybuild/easyconfigs/m/mock/mock-1.0.1-foss-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'mock'
+version = '1.0.1'
+
+homepage = 'https://pypi.python.org/pypi/mock'
+description = """A Python Mocking and Patching Library for Testing.
+ mock is a library for testing in Python.
+ It allows you to replace parts of your system under test
+ with mock objects and make assertions about how they have been used."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://pypi.python.org/packages/source/%s/%s/' % (name[0],name)]
+sources = [SOURCE_TAR_GZ]
+
+python = "Python"
+pythonversion = '2.7.9'
+#mock is part of Python 3.3
+pyshortver = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
```

Comment:
this has no other dependencies except for Python, so why not include it in the Python easyconfig as an extension?

any particular reason?